src/components/dialogmote/innkalling/BehandlerRadioGruppe.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from "react";
+import { Radio, RadioGruppe } from "nav-frontend-skjema";
+import styled from "styled-components";
+import {
+  BehandlerDialogmeldingDTO,
+  behandlerOneliner,
+} from "@/data/behandlerdialogmelding/BehandlerDialogmeldingDTO";
+
+const texts = {
+  behandlerLegend: "Behandler som inviteres til dialogmøtet",
+  behandlerInfo:
+    "Fastlegen vil få en dialogmelding med invitasjon. Dersom du ønsker å invitere en annen behandler, må du fortsatt sende innkallingen fra Arena.",
+  noBehandler: "Ingen behandler",
+};
+
+const SpaceyRadioGruppe = styled(RadioGruppe)`
```

Comment:
Vet ikke om denne komponenten er så space'a at den trenger å hete spacey? 😁  Pleier vi ikke å bare prefix'e med Styled?

Reply:
Den trenger jo litt ekstra plass da 🤔 

Pleier og pleier 🤷 Jeg har prøvd å unngå det hvis jeg kan finne på et bedre navn. At noe er "Styled" betyr ingenting, så jeg liker det egentlig ikke som prefix. Problemet for meg har vært at vi ofte har "brukt opp" det gode, forklarende navnet på den komponenten som skal styles. (Og hvis det er mye styling, kan vi jo ikke prefixe med alle de ulike tingene som gjøres.)

Reply:
Jeg er egentlig enig. Det blir utfordrende når design systemet kaprer de gode navnene og man blir nødt til å legge på luft i etterkant. 🤔